dmutils/request_id.py

```diff
@@ -1,50 +1,104 @@
 import uuid
+from itertools import chain
 
 from flask import request, current_app
-from flask.wrappers import Request
 
 
-class CustomRequest(Request):
-    _request_id = None
-
+class RequestIdRequestMixin(object):
```

Comment:
Feels like a class docstring would perhaps be the place to link to a page that defines trace ID and span ID?

Reply:
Yup.